content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,212 @@
+      The value of this variable cannot be greater than 25 characters. It must
+      contain just a name, not a directory path.
+      This value also becomes the name of the directory where your Kubeflow
+      configurations are stored, that is, the Kubeflow application directory.
+
+    * **${KF_DIR}** - The full path to your Kubeflow application directory.
+
+1. Configure OIDC Auth service settings:
+
+   In `/manifests/stacks/azure/application/oidc-authservice/kustomization.yaml` update the settings with values corresponding your app registration as follows:
+
+    ```
+    - client_id=<client_id>
+    - oidc_provider=https://login.microsoftonline.com/<tenant_id>/v2.0
+    - oidc_redirect_uri=https://<load_balancer_ip> or dns_name>/login/oidc
+    - oidc_auth_url=https://login.microsoftonline.com/<tenant_id>/oauth2/v2.0/authorize
+    - application_secret=<client_secret>
+    - skip_auth_uri=
+    - namespace=istio-system
+    - userid-header=kubeflow-userid
+    - userid-prefix=
+    ```
+
+1. Configure OIDC scopes:
+
+    In `/manifests/istio/oidc-authservice/base/statefulset.yaml` update [OIDC scopes](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes) to remove groups and keep profile and email.
+
+    ```
+    - name: OIDC_SCOPES
+     value: "profile email"
+    ```
+
+3. Deploy Kubeflow:
+
+    ```
+    kfctl apply -V -f ${CONFIG_URI}
+    ```
+
+4. Check that the resources were deployed correctly in namespace `kubeflow`:
+
+    ```
+    kubectl get all -n kubeflow
+    ```
+
+## Expose Kubeflow securely over HTTPS
+
+1. Update Istio Gateway to expose port 443 with HTTPS and make port 80 redirect to 443:
+
+    ```
+    kubectl edit -n kubeflow gateways.networking.istio.io kubeflow-gateway
+    ```
+
+    The Gateway spec should look like the following:
+
+    {{< highlight yaml >}}
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+    name: kubeflow-gateway
+    namespace: kubeflow
+    spec:
+    selector:
+        istio: ingressgateway
+    servers:
+    - hosts:
+        - '*'
+        port:
+        name: http
+        number: 80
+        protocol: HTTP
+              # Upgrade HTTP to HTTPS
+        tls:
+        httpsRedirect: true
+    - hosts:
+        - '*'
+        port:
+        name: https
+        number: 443
+        protocol: HTTPS
+        tls:
+        mode: SIMPLE
+        privateKey: /etc/istio/ingressgateway-certs/tls.key
+        serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+    {{< /highlight >}}
+
+1. Expose Kubeflow with a load balancer service:
+
+    To expose Kubeflow with a LoadBalancer service, change the type of the `istio-ingressgateway` service to `LoadBalancer`.
+
+    ```
+    kubectl patch service -n istio-system istio-ingressgateway -p '{"spec": {"type": "LoadBalancer"}}'
+    ```
+
+   After that, obtain the LoadBalancer's IP or Hostname from its status and create the necessary certificate.
```

Comment:
As per the Microsoft [docs](https://docs.microsoft.com/en-us/azure/architecture/guide/technology-choices/load-balancing-overview), since you're using the service name, let's add backticks (code font) here:

```suggestion
   After that, obtain the `LoadBalancer` IP or Hostname from its status and create the necessary certificate.
```